scripts/ebeam_scan_to_multiprobe.py

```diff
@@ -7,36 +7,56 @@
 
 Copyright © 2019 Thera Pals, Delmic
 
-This script provides a command line interface for aligning the ebeam scan to multiprobe.
+This script provides a command line interface for aligning the ebeam scan to
+multiprobe.
+
+This script implements the following steps:
+* Acquire image of multiprobe to determine angle of multiprobe.
+* Move multiprobe on single axis with deflectors (by changing DC offset on scan
+deflector AWG) to obtain angle of movement.
+* Calculate angle of movement relative to angle of multiprobe.
+* Match angle of movement to angle of multiprobe by changing scan rotation in
+XT/SEM PC.
 
 """
 from __future__ import division
 
 import argparse
 import logging
 import math
-import numpy
 import sys
 from builtins import input
 
+import numpy
+
+sys.path.insert(1, "../../sonic-engineering/scanwaveform")
+import dcoffset
+
 from odemis import model
 from odemis.acq.align.spot import FindGridSpots
 from odemis.util.driver import get_backend_status, BACKEND_RUNNING
 
 
 def get_deflection_angle(coordinates):
     """
-    Fit a line through a set of coordinates and calculate the angle between that line and the x-axis. This angle is
-    called the deflection angle.
-
-    coordinates (ndarray of shape nx2): (x, y) coordinates of center positions of the grid of spots.
-    return:
-        phi (float): deflection angle in radians.
+    Fit a line through a set of coordinates and calculate the angle between that
+    line and the x-axis. This angle is called the deflection angle.
+
+    Parameters
+    ----------
+    coordinates: (ndarray of shape nx2)
+        (x, y) coordinates of center positions of the grid of spots.
```

Comment:
When I first read it I thought it is the center of the grid, but that didn't match with the type. Maybe:
`(x, y) coordinates of the center positions of each spot in the grid.`

Reply:
It's the center positions of the moved grid. I will change the text, is this clear:
`(x, y) coordinates of the center positions of the moved grid of spots.`